src/modules/home/footer/Footer.tsx

```diff
@@ -5,6 +5,8 @@ import SendIcon from '@material-ui/icons/Send';
 import { Logo, Button } from 'ui';
 
 import csx from './Footer.scss';
+import { NavLink } from 'react-router-dom';
+import { Guard } from 'core/auth';
```

Comment:
This import should be placed after react-stuff dependencies + enter + import here + enter + import csx - check other files for reference